dev/com.ibm.websphere.org.reactivestreams.reactive-streams.1.0.2/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR. From this we consume both the API and TCK
+#
+# Use com.ibm.ws.org.reactivestreams.interfaces bundle for API
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0.2
```

Comment:
The bundle version should be 1.0; this is the first version of this bundle. The fact that it contains version 1.0.2 of the API is irrelevant.